TRACING.md

```diff
@@ -116,6 +115,21 @@ Each trace mode is sub-divided into two different types:
 | isolated | The traces only apply to the agent; after the container has been destroyed, the first span will start at agent startup and the last at agent shutdown | Observing agent lifespan. | |
 | collated | In this mode, spans are associated with their `kata-runtime` initiated counterparts. | Understanding how the runtime calls the agent. | Requires runtime tracing to be enabled in `configuration.toml` (`enable_tracing=true`). |
 
+# Agent shutdown behaviour
+ 
+As shown in [Trace modes](#trace-modes) section, the different trace modes
```

Comment:
Add "the": 
As shown in the [Trace modes](#trace-modes) section, the different trace modes